src/main/java/com/amazonaws/services/dynamodbv2/AcquireLockOptions.java

```diff
@@ -317,7 +333,7 @@ public AcquireLockOptions build() {
             }
             return new AcquireLockOptions(this.partitionKey, this.sortKey, this.data, this.replaceData, this.deleteLockOnRelease, this.acquireOnlyIfLockAlreadyExists,
                     this.refreshPeriod, this.additionalTimeToWaitForLock, this.timeUnit, this.additionalAttributes, sessionMonitor,
-                    this.updateExistingLockRecord, this.shouldSkipBlockingWait, this.acquireReleasedLocksConsistently);
+                    this.updateExistingLockRecord, this.shouldSkipBlockingWait, this.acquireReleasedLocksConsistently, reentrant);
```

Comment:
```suggestion
                    this.updateExistingLockRecord, this.shouldSkipBlockingWait, this.acquireReleasedLocksConsistently, this.reentrant);
```

- consistency

Reply:
Also update the toString() below

Reply:
Addressed review comments in latest commit.